nca/NetworkConfig/LiveSim/dns_pods.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-system
+  labels:
+    name: kube-system
```

Comment:
This label should be changed to: `kubernetes.io/metadata.name: kube-system`

Reply:
kube-system can be referred in several ways.
Added this label as well.